index.html

```diff
@@ -1105,7 +1105,7 @@ <h2 property="schema:name">Discovery</h2>
                 <li><a href="https://solid.github.io/type-indexes/">Type Index Documents</a>, containing links to specific resource types.</li>
                 <li><a href="#extended-profile">Extended Profile Documents</a>, containing supplementary information about the WebID owner.</li>
               </ul>
-              <p>A profile can be compiled by reading all statements with the WebID as the subject, regardless of their originating
+              <p>A profile can be compiled by reading all statements with the WebID as the subject or the object, regardless of their originating
```

Comment:
I don't think you're actually addressing any "reverse use of properties".

Whether WebIDs are within the Domain or Range of a predicate determines whether a WebID might be found in the Subject or Object position, respectively, of a triple containing that predicate.

An inverse predicate may exist, that has a Domain and Range that are the Range and Domain, respectively, of another predicate.

A simple implementation of the (insufficiently specified) SPARQL verb `DESCRIBE` may be written as `SELECT * WHERE { { ?s ?p example:WebID } UNION { example:WebID ?p ?o } }`.

That said, I think that a profile cannot be assembled (nor compiled) by reading only statements where the WebID is the subject, nor by reading only statements where the WebID is the object; such manual assembly MUST read all statements where the WebID is *either* the subject or the object.

```suggestion
              <p>A profile can be assembled by collecting all statements that have the WebID as the subject or the object, regardless of their originating
```